src/_posts/databases/2000-01-01-backup-policies.md

```diff
@@ -0,0 +1,146 @@
+Starter and Business plans. Available for all supported databases, scheduled
+backups are performed every 24 hours. The start time can be customized through
+the dashboard or via the API, enabling you to align backups with your workload
+and core business operations.
+
+#### Retention Policy for Scheduled Backups
+
+We retain a limited number of backups based on your database plan:
+
+- **Daily backups**: one backup is retained for each of the last 7 days, for a
+  total of 7 backups.
+- **Weekly backups**: the daily backup taken on Sunday is retained as the
+  weekly backup.
+- **Monthly backups**: the daily backup from the first Sunday of the month is
+  retained as the monthly backup.
+
+| Plan     | Daily Backups  | Weekly Backups  | Monthly Backups   |
+| -------- | -------------- | --------------- | ----------------- |
+| Sandbox  | None           | None            | None              |
+| Starter  | 7 rolling days | 4 rolling weeks | None              |
+| Business | 7 rolling days | 8 rolling weeks | 12 rolling months |
+
+Deleting a database doesn’t immediately remove its scheduled backups. They
+expire gradually according to their retention policy.
+
+### Manual Backups
+
+Scalingo provides a manual backup service for databases in the Starter and
+Business plans. Available for all supported databases, this feature lets you
+create a backup on demand, such as before an upgrade or any important operation
+on your data.
+
+#### Retention Policy for Manual Backups
+
+The number of manual backups you can store is limited by your plan (see below).
+Manual backups are kept for an unlimited amount of time, as long as the number
+of backups doesn't exceed the plan's limit. If creating a new backup exceeds
+this limit, the oldest backup is automatically deleted.
+
+| Plan     | Backups Retained        |
+| -------- | ----------------------- |
+| Sandbox  | None                    |
+| Starter  | Last 10 rolling backups |
+| Business | Last 50 rolling backups |
+
+When a database is deleted, a retention policy is retroactively applied to
+all existing manual backups, based on their creation date. The retention
+duration depends on the plan:
+
+| Plan     | Retention Duration |
+| -------- | ------------------ |
+| Sandbox  | None               |
+| Starter  | 1 month            |
+| Business | 1 year             |
+
+If manual backups have already exceeded their retention period, they are
+deleted immediately.
+
+
+## Point-in-Time-Recovery
+
+Point-in-Time Recovery (PiTR) allows the restoration of your PostgreSQL®
+database to any specific point in time within the last 7 days. This feature is
+available for Starter and Business plans and provides an additional layer of
+data protection by keeping continuous backups of your database's state.
+
+{% note %}
+Point-in-Time Recovery (PITR) technology operates independently of the
+database's state, load, or size, making it a reliable solution for handling
```

Comment:
It's a bit hard to solve this one:
- we are in the Backup Policies page, so I'd like to avoid putting technical stuff in there,
- Also the goal is to have a dedicated page covering this in each Database sub-section, like this one: https://doc.scalingo.com/databases/postgresql/backing-up - So I can't put a generic link in here :-/

Any idea? :thinking:

Reply:
Indeed!